cdcpd/include/cdcpd/cdcpd.h

```diff
@@ -68,6 +68,25 @@ enum OutputStatus {
   Success,
 };
 
+class CdcpdParameters
```

Comment:
Might be nice to keep all caps, to be consistent with CDCPD class.

Reply:
Got it. Addressed in the most recent commit